admin-dev/themes/new-theme/js/pages/order/create/create-order-page.js

```diff
@@ -24,18 +24,18 @@
  */
 
 import createOrderPageMap from './create-order-map';
-import CustomerSearcherComponent from './customer-searcher-component';
+import CustomerManager from './customer-manager';
```

Comment:
Idea from @PierreRambaud : manage this dependency list using alphabetical order. This helps avoiding git conflicts as when 2 PRs modify this list, they insert new items at different locations instead of always at the end of the list